src/main/java/vampireNumbers/VampireNumbersDefinition.java

```diff
@@ -0,0 +1,43 @@
+package vampireNumbers;
+
+import org.slf4j.*;
+
+import java.util.*;
+
+public class VampireNumbersDefinition {
+    private static final Logger LOGGER = LoggerFactory.getLogger(VampireNumbersDefinition.class);
+    private boolean isVampireResult;
+
+    private boolean getFangs(char[] value, int initialValue) {
+        Permutation.permute(value);
+        Set<String> permutedNumbers = Permutation.result;
+        Iterator iterator = permutedNumbers.iterator();
+        while (iterator.hasNext()) {
+            String[] ab = iterator.next().toString().split("");
+            String[] a = {ab[0], ab[1]};
+            String[] b = {ab[2], ab[3]};
+            if ((a[1].equals("0") && b[1].equals("0")) || (a[0].equals("0") || b[0].equals("0"))) {
```

Comment:
i believe it could be simplified

Reply:
No correction is required to '(a[1].equals("0") && b[1].equals("0")) || (a[0].equals("0") || b[0].equals("0"))'.
 
a[1].equals("0") && b[1].equals("0") - covers the cases like (10, 20), (50, 60),..., so that the cases when both numbers ends with '0' are excluded.
(a[0].equals("0") || b[0].equals("0")) -covers the cases like (06, 17), (99, 01), ..., so that only double digits are included.